src/main/java/com/zaxxer/influx4j/InfluxDB.java

```diff
@@ -204,6 +229,26 @@ public void query(final Query query, final Writer writer) {
       try {
          final String q = "db=" + query.getDatabase() + "&q=" + query.getCommandWithUrlEncoded();
 
+         executeQuery(q, writer);
+      } catch (final Exception e)
+      {
+         throw new RuntimeException(e);
+      }
+   }
+
+   /**
+    * Execute a {@link Query}, with the result JSON being written to the speicifed {@link Writer}.
```

Comment:
Similar description to above, indicating that the timestamps will be in Unix epoch format.